cms/templates/js/container-unit-access.underscore

```diff
@@ -0,0 +1,15 @@
+<%
+var selectedGroupsLabel = userPartitionInfo['selected_groups_label'];
+var selectedPartitionIndex = userPartitionInfo['selected_partition_index'];
+%>
+<% if (selectedGroupsLabel) { %>
+    <small class="access-message"><%-
```

Comment:
@bbaker6225 <small> tag should be reserved for things like: smallprint, legal text, etc. You should use CSS to make the font size smaller.
FYI @cahrens